scripts/migrate-test-names.mjs

```diff
@@ -21,9 +21,9 @@ const usage = `
 Run this script in a package to convert its test names and Ava config.
 
 To run it over all packages,
-  yarn lerna exec --concurrency 1 --no-bail '../../scripts/migrate-test-names.mjs'
+  yarn workspaces foreach -A exec '../../scripts/migrate-test-names.mjs'
```

Comment:
we could also remove this file because the migration is done.

Reply:
i considered it but since theres other similar migration files, ill leave removal to a separate PR